pkg/cluster/cluster.go

```diff
@@ -461,23 +477,32 @@ func (c *Cluster) updateStatus() error {
 	return nil
 }
 
-func (c *Cluster) reportFailedStatus() error {
+func (c *Cluster) reportRunningStatus() {
+	c.reportPhaseStatus(spec.ClusterPhaseRunning)
```

Comment:
this is not what i want. in the normal phase, our main routine should drive all the retry. there should not be any inner retry loop.